src/Features/Core/Portable/InheritanceMargin/AbstractInheritanceMarginService_Helpers.cs

```diff
@@ -387,25 +361,23 @@ internal abstract partial class AbstractInheritanceMarginService
             Solution solution,
             INamedTypeSymbol interfaceSymbol,
             int lineNumber,
-            ImmutableArray<ISymbol> baseSymbols,
-            ImmutableArray<ISymbol> derivedTypesSymbols,
+            ImmutableArray<SymbolGroup> baseSymbolGroups,
+            ImmutableArray<SymbolGroup> derivedTypesSymbolGroups,
             CancellationToken cancellationToken)
```

Comment:
aside, this member returns an `Item?`.  is null actually possible to return here?

Reply:
https://sourceroslyn.io/#Microsoft.CodeAnalysis.Features/InheritanceMargin/InheritanceMarginItem.cs,8402ab3583b2cc9e,references
It returns Item?